### PR TITLE
Updates issue template, notes on README.me and fixes line ending

### PR DESCRIPTION
**Pull request by DRSDavidSoft**
1. Update the issue template: removed broken links, add ConEmu documentation link
2. Notes on README.me: refined some notes for clarity, added info on WSL, VS Code & Hyper.
3. Fix line ending: a .rc file line ending has been fixed.

### CODE REVIEW

**Comment by Stanzilla** [comment]
I think it would be best if you did it in one commit, so you don't add content here and remove it in the next one.

**Comment by DRSDavidSoft** [comment]
@Stanzilla Is it approved?

File: .github/issue_template.md [hunk 1]
```diff
@@ -64,7 +64,9 @@
   - Post a screenshot or the command the triggered the problem, if applicable.
   - Avoid using ambiguous phrases like: doesn't work, there'a problem, etc.
     Help us reproduce the issue by explaining what went wrong, and what did you expect to happen.
-  - Read the following page to avoid posting a bad issue: https://conemu.github.io/en/BadIssue.html
+  - We at Cmder only understand the English language.
```
Comment (Stanzilla) [inline .github/issue_template.md]: That sounds a bit rude :D "Please keep ticket language to English" or so?

Comment (DRSDavidSoft) [reply .github/issue_template.md]: That sounded a bit unprofessional! I changed it.

**Comment by DRSDavidSoft** [comment]
@Stanzilla changed as requested.

**Pull request merged**